index.bs

```diff
@@ -258,36 +258,38 @@ interface ModelBuilder {
 ### batchNormalization ### {#api-modelbuilder-batchnorm}
 Normalize the tensor values across dimensions in a batch through a specialized [[BatchNorm]] [transform](https://en.wikipedia.org/wiki/Batch_normalization#Batch_Normalizing_Transform). The *mean* and *variance* tensors are previously calculated during model training pass.
 <script type=idl>
+dictionary BatchNormalizationOptions {
+  Operand scale;
+  Operand bias;
+  long axis = 1;
+  float epsilon = 1e-5;
+};
+
 partial interface ModelBuilder {
-  Operand batchNormalization(Operand input, Operand mean, Operand variance, 
-                             optional Operand scale, optional Operand bias, 
-                             optional long axis = 1, optional float epsilon = 1e-5);
+  Operand batchNormalization(Operand input, Operand mean, Operand variance,
+                             optional BatchNormalizationOptions options = {});
```

Comment:
The usage of `scale`, `bias`, `axis` and `epsilon` in the sample code of `batchNormalization` would be changed to `options.scale`, `options.bias`, `options.axis` and `options.epsilon`.

Reply:
Do you mean samples in webnn_samples repo? I believe I've updated all the samples in this spec and in the explainer to reflect this change. The webnn_samples change would be on a separate PR to that repo.

Reply:
I mean the sample/pseudo code of batchNormalization in the spec
```js
let shape = [1,-1,1,1];
return builder.add(
    builder.mul(
      builder.reshape(scale, shape),
      builder.div(
        builder.sub(input, builder.reshape(mean, shape)),
        builder.sqrt(builder.add(builder.reshape(variance, shape), builder.constant(epsilon)))
        )
      ),
    builder.reshape(bias, shape)
  );
```
Once the spec change landed, I can take care of the update of webnn-polyfill and webnn-samples.

Reply:
Fixed.

Reply:
BTW, can we just delete index.html from the repo since we already build it off index.bs? That way we don't need to keep updating the file manually for every change we make.

Reply:
+1. We can use github actions to build the index.html and deploy it to gh-pages branch.

Reply:
Created #113 for this. Please take a look.

Reply:
Thanks @huningxin